src/services/authService.js

```diff
@@ -50,6 +50,7 @@ module.exports = () => {
           () => '?',
         )})`,
         whereStringValues: [...roleIds],
+        limit: 10000,
```

Comment:
I know why you did this, but I'm a little bit hesitant about it. I'm thinking of a way of bypassing the pagination instead of '/rows' endpoints, by providing a query param for example.

Reply:
Sure, but here we are using the `rowService` we built and if we update the `rowService.get()` function we have to also modify all of the other files that are calling this function, shoud we rebuild the `rowService.get()` function so it can be better?

Reply:
As a _slight_ compromise, we could reduce the limit to `100` - I know that would still meet my requirements.  I _guess_ that if there are more than the `limit` the `next` URL will allow the client to page through the content.